src/core/prototypes/controls.ts

```diff
@@ -211,6 +211,8 @@ export interface ArrayWidgetOptions {
 export interface ScrollListOptions {
   height?: number;
   maxHeight?: number;
+  paddingBottom?: number;
```

Comment:
Should `paddingBottom` and `paddingTop` be included in a style definition rather than the interface for a control interface?

Reply:
Updated. Added `styles` argument